src/main/java/com/foundation/view/FormEvent.java

```diff
@@ -0,0 +1,35 @@
+package com.foundation.view;
+
+import java.util.EventObject;
+
+public class FormEvent extends EventObject {
+
+    private String one;
```

Comment:
Improve the name of this variables, "one" and "two" do not give a good reference of purpose of them